webrender/src/renderer.rs

```diff
@@ -564,6 +564,11 @@ struct SourceTextureResolver {
     /// The current cache textures.
     cache_rgba8_texture: Option<Texture>,
     cache_a8_texture: Option<Texture>,
+
+    pass_rgba8_textures: FastHashMap<usize, usize>,
```

Comment:
We could use a stronger type here, or at least a comment to describe what this is mapping between.